pages/wishlist.html

```diff
@@ -0,0 +1,242 @@
+          <div class="flex mb-0-5">
+            <span class="card-discount-text mb-0-5 font-semibold">50% off</span>
+            <span class="card-shipment font-semi">Ships in <b> 1 day</b></span>
+          </div>
+          <button class="card-buy-btn">Add to Cart</button>
+        </section>
+      </article>
+
+      <!-- Card with Text Overlay -->
+      <article class="card-verticle flex flex-column mr-2 my-1">
+        <div class="overlay-container">
+          <section class="image-container mb-0-5">
+            <img
+              loading="lazy"
+              class="card-image responsive-img"
+              src="../assets/images/sofa.jpg"
+              alt="sofa-ikea"
+            />
+            <span class="text-badge">Best Seller</span>
+            <i class="fa-regular fa-circle-xmark dismiss-btn"></i>
+            <i class="fa-regular fa-heart icon-badge"></i>
+            <div class="card-rating flex flex-center">
+              <span class="card-rating-star">4.5</span>
+              <i class="fa-solid fa-star rating-star-icon"></i>
+            </div>
+            <div class="card-addtocart orange-btn hide">
+              <a href="#" class="add-to-cart-btn">Add To Cart</a>
+            </div>
+          </section>
+          <section class="px-0-5 mb-0-5">
+            <p class="card-title font-semi mb-0-5">
+              Andrea Sofa 3 seater in peacock green color
+            </p>
+            <span class="card-brand font-semi mb-1">Ikea</span>
+            <div class="card-price mb-0-5">
+              <span class="card-price-offer font-bold">₹ 39,999</span>
+              <span class="card-price-mrp font-semibold">₹ 79,999</span>
+            </div>
+            <div class="flex mb-0-5">
+              <span class="card-discount-text mb-0-5 font-semibold"
+                >50% off</span
+              >
+              <span class="card-shipment font-semi"
+                >Ships in <b> 1 day</b></span
+              >
+            </div>
+            <button class="card-buy-btn">Add to Card</button>
+          </section>
+          <div class="overlay-text">
+            <div class="overlay-text-container">Out Of Stock</div>
+          </div>
+        </div>
+      </article>
+
+      <!-- Card three -->
+      <article class="card-verticle flex flex-column mr-2 my-1">
+        <section class="image-container mb-0-5">
+          <img
+            loading="lazy"
+            class="card-image responsive-img"
+            src="../assets/images/sofa.jpg"
+            alt="sofa-ikea"
+          />
+          <span class="text-badge">Best Seller</span>
+          <i class="fa-regular fa-circle-xmark dismiss-btn"></i>
+          <i class="fa-regular fa-heart icon-badge"></i>
+          <div class="card-rating flex flex-center">
+            <span class="card-rating-star">4.5</span>
+            <i class="fa-solid fa-star rating-star-icon"></i>
+          </div>
+          <div class="card-addtocart orange-btn hide">
```

Comment:
Follow proper class name i.e here addtocart should have hypen between them.

Reply:
yes I will make changes to this. Thanks for pointing this issue